src/main/java/frc/trigon/robot/motorsimulation/MotorSimulationConfiguration.java

```diff
@@ -0,0 +1,19 @@
+package frc.trigon.robot.motorsimulation;
+
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+
+public class MotorSimulationConfiguration {
```

Comment:
wdym conversions factor?

Reply:
What are the units that are going to be returned (i.e. 360 per rotation)

Reply:
so do I need to make an enum for the unit type? and I don't understand what u mean with "whether the pid should be with continuous output."

Reply:
1. No need for an enum, this will just be using a number. Look at `sparkMax.setConversionsFactor()`
2. Look at `pidController.enableContinuousOutput` and understand what it does